linera-chain/src/unit_tests/chain_tests.rs

```diff
@@ -246,20 +246,30 @@ async fn test_application_permissions() {
     let valid_block = make_first_block(chain_id)
         .with_incoming_bundle(bundle)
         .with_operation(app_operation.clone());
-    let outcome = chain.execute_block(&valid_block, time, None).await.unwrap();
+    let outcome = chain.execute_block(&valid_block, time, None).await?;
     let value = HashedCertificateValue::new_confirmed(outcome.with(valid_block));
 
     // In the second block, other operations are still not allowed.
-    let invalid_block = make_child_block(&value.clone().try_into().unwrap())
-        .with_simple_transfer(chain_id, Amount::ONE)
-        .with_operation(app_operation.clone());
+    let invalid_block = make_child_block(
+        &value
+            .clone()
+            .try_into()
+            .map_err(|error| anyhow!("{error}"))?,
```

Comment:
Unfortunately not, because the `Err` variant in this case is `str`, which doesn't implement `std::error::Error`. I opened #2951 with a proposal for an alternative.